.github/workflows/test-install.yml

```diff
@@ -45,9 +45,6 @@ jobs:
     - name: Install utils/ dependencies
       run: uv pip install --system -r utils/requirements.txt
 
-    - name: Validate uv lockfile
```

Comment:
I think it's better to have this check only in one place.

Reply:
why do we use here latest and not version `0.5.22`?